x-pack/plugins/timelines/public/components/t_grid/helpers.tsx

```diff
@@ -8,14 +8,15 @@
 import type { Filter, EsQueryConfig, Query } from '@kbn/es-query';
 import { isEmpty, get } from 'lodash/fp';
 import memoizeOne from 'memoize-one';
+import { ALERT_WORKFLOW_STATUS } from '@kbn/rule-data-utils';
 import {
   elementOrChildrenHasFocus,
   getFocusedAriaColindexCell,
   getTableSkipFocus,
   handleSkipFocus,
   stopPropagationAndPreventDefault,
 } from '../../../common';
-import type { IIndexPattern } from '../../../../../../src/plugins/data/public';
+import { esFilters, IIndexPattern } from '../../../../../../src/plugins/data/public';
```

Comment:
Isn't `esFilters` deprecated and there's notes about the non-deprecated version we are to use now?

```ts
 * Filter helpers namespace:
 * @deprecated Import helpers from the "@kbn/es-query" package directly instead.
 * @removeBy 8.1
```